app/static/src/app/components/options/ParameterSetView.vue

```diff
@@ -97,18 +125,54 @@ export default defineComponent({
             store.commit(`run/${RunMutation.ToggleParameterSetHidden}`, props.parameterSet.name);
         };
 
+        const paramNameInput = ref(null);
+        const editDisplayName = ref(false);
+        const newDisplayName = ref(props.parameterSet.displayName);
+        const editDisplayNameOn = () => {
+            editDisplayName.value = true;
+            nextTick(() => {
+                (paramNameInput.value! as HTMLInputElement).select();
+            });
+        };
+        const saveDisplayName = () => {
+            const payload = {
+                parameterSetName: props.parameterSet.name,
+                newDisplayName: newDisplayName.value
+            };
+            store.commit(`run/${RunMutation.SaveParameterDisplayName}`, payload);
+            if (!props.parameterSet.isDisplayNameError) {
+                editDisplayName.value = false;
+            }
+        };
+
+        const turnOffDisplayNameError = () => {
+            if (props.parameterSet.isDisplayNameError) {
+                store.commit(`run/${RunMutation.TurnOffDisplayNameError}`, props.parameterSet.name);
+            }
+        };
+
+        watch(newDisplayName, turnOffDisplayNameError);
```

Comment:
added that into the `SaveParameterDisplayName`, the reason there is a special mutation turn off error just for turning off error is because it is probably better to not do all the validation from `SaveParameterDisplayName` for every letter the user inputs!